File: spec/code_samples/Python python3/wallets@{walletId}/get.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("GET", "/wallets/%7BwalletId%7D")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node unirest/wallets@{walletId}/get.js
var unirest = require("unirest");

var req = unirest("GET", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/wallets/%7BwalletId%7D");


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
